### ISSUE TITLE
Staging environment for tutorials

### ISSUE THREAD

**Issue by timClicks** [canonical-web-and-design/juju.is#52]
Is it possible to set up a staging environment for viewing how draft tutorials are rendered on juju.is/tutorials before they're published?

**Comment by anthonydillon** [comment]
@nottrobin Do you think we could get staging to pull all tutorials including draft ones?

**Comment by nottrobin** [comment]
@anthonydillon does Discourse have a concept of Drafts? Where is it? I can't see it at first glance. Or do you mean that we could maybe allow the loading of any Discourse topic before it's officially added to the index topic or something?

**Comment by anthonydillon** [comment]
@nottrobin Yeah your right, this may need to be developed. Could we add a section of drafts to the index and is only visible with the URL to the tutorial but not indexed or listed?

**Comment by Lukewh** [comment]
Could we do something like `https://juju.is/docs/drafts/<discourse topic path>`?

**Comment by nottrobin** [comment]
Yeah that should be possible. The same pattern could be applied for docs.

An issue is that the index topic is actually also how we find tutorials - as in the way we know that any given discourse topic is supposed to be a tutorial is through whether it's included in the index topic.

Drafts like this would have to be category-based I guess?

``` python3
        parser=TutorialParser(
            api=DiscourseAPI(
                base_url="https://discourse.charmhub.io/",
                session=session,
                api_key=DISCOURSE_API_KEY,
                api_username=DISCOURSE_API_USERNAME,
                get_topics_query_id=2,
            ),
            index_topic_id=2628,
            url_prefix=url_prefix,
            drafts_category="tutorials"
        ),
```

We should probably write a spec for this.
